.monorepolint.config.ts

```diff
@@ -249,6 +265,14 @@ module.exports = {
           dependencies: {
             tslib: "^2.3.0",
           },
+          devDependencies: {
+            nyc: "*",
+          },
+        },
+        includePackages: [...TS_PACKAGES, ...JS_PACKAGES],
```

Comment:
This got included in a weird way. tslib dependency should only apply to TS_PACKAGES, I guess just move the above dependencies block to the next entry? It also wound up getting added to all the package.json files so that'll need to get pulled back out. I think you can do that with monorepolint by setting an explicit removal for the JS_PACKAGES on tslib dependency and then running monorepolint and then deleting the rule before committing.

Reply:
Good spot, sorry about that. I figured out how to remove it; you can do:

```
    ":banned-dependencies": {
      options: {
        bannedDependencies: ["tslib"],
      },
      includePackages: [...JS_PACKAGES],
    },
 ```
 
 For future reference